firehose.go

```diff
@@ -143,6 +143,6 @@ func (p *Producer) sendFirehoseRecords(args *gokinesis.RequestArgs) {
 	}
 
 	if conf.Debug.Verbose && p.getMsgCount()%100 == 0 {
-		log.Println("Messages sent so far: " + strconv.Itoa(p.getMsgCount()))
+		log.Println("Messages sent so far: " + strconv.FormatInt(p.getMsgCount(), 10))
```

Comment:
From the docs `Itoa` appears to be shorthand for `Itoa is shorthand for FormatInt(int64(i), 10)`. Do we need the longhand here?